jboss-eap-clustered-multivm-rhel/scripts/jbosseap-setup-redhat.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/sh
+
+adddate() {
+    while IFS= read -r line; do
+        printf '%s %s\n' "$(date "+%Y-%m-%d %H:%M:%S")" "$line";
+    done
+}
+
+echo "Red Hat JBoss EAP Cluster Intallation Start " | adddate >> jbosseap.install.log
+/bin/date +%H:%M:%S  >> jbosseap.install.log
+
+echo 'export EAP_HOME="/opt/rh/eap7/root/usr/share"' >> ~/.bash_profile
+source ~/.bash_profile
+touch /etc/profile.d/eap_env.sh
+echo 'export EAP_HOME="/opt/rh/eap7/root/usr/share"' >> /etc/profile.d/eap_env.sh
+
+while getopts "a:t:p:f:" opt; do
```

Comment:
why do we have this for only some of the parameters?  some are positional, some not...

Reply:
This is to download the WAR file the same way we handle the script itself - by downloading from the staged location. I have followed the same method from https://github.com/Azure/azure-quickstart-templates/blob/master/100-marketplace-sample/scripts/copyfilefromazure.sh as you mentioned earlier for the WildFly on CentOS template https://github.com/Azure/azure-quickstart-templates/pull/7961#discussion_r472398489